app/assets/stylesheets/tylium/modules/_activities.scss

```diff
@@ -39,3 +47,104 @@
     }
   }
 }
+
+// this is for activities#index
+.all-activities-container {
+
+  .activity-date-header {
+    display: flex;
+    justify-content: center;
+
+    time {
+      background-color: $primaryBgColor;
+      font-size: 1.5rem;
+      padding: 1rem 0;
+    }
+  }
+
+  .activities-group {
+    width: 50%;
+
+    .activity {
+      border: 1px solid transparent;
+      padding: 1rem;
+      transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out;
```

Comment:
Do we have a standard transition that gets used, and should it become a variable if yes?